### PR TITLE
LC: fix nil pointer dereference bug

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

File: pkg/localcontroller/manager/incrementallearningjob.go [hunk 1]
```diff
@@ -192,7 +192,7 @@ func (im *IncrementalJobManager) evalTask(job *IncrementalLearningJob) error {
 	if currentType == sednav1.ILJobStageCondWaiting {
 		err := im.loadDeployModel(job)
 		if err != nil {
-			klog.Warningf("job(name=%s) failed to sync deploy model, and waiting it: %v",
+			return fmt.Errorf("job(name=%s) failed to sync deploy model, and waiting it: %v",
```
Comment (llhuii) [inline pkg/localcontroller/manager/incrementallearningjob.go:195]: it does not need job id info into `error` that leading to redundant message.

Comment (JimmyYang20) [reply pkg/localcontroller/manager/incrementallearningjob.go]: ok! fix it. same as trainTask

**Pull request merged**